R/fcn_computeQC.R

```diff
@@ -121,7 +121,7 @@ if (enabled_parameters)
   ## seperate FASTA files (usually they destroy the layout)
   idx_fastafile = grepl("fasta file", d_parAll$parameter, ignore.case = T)
   d_par_file = d_parAll[idx_fastafile, ]
-  fasta_files = sapply(unlist(strsplit(d_par_file$value, "\n")), function(x) rev(strsplit(x,"\\", fixed=T)[[1]])[1])
+  fasta_files = sapply(unlist(strsplit(d_par_file$value, "\n")), function(x) rev(strsplit(x,.Platform$file.sep, fixed=T)[[1]])[1])
```

Comment:
it will run fine, but I guess you'll see the full path to the FASTA 
files in on the first page of the report (instead of just the basename).

Am 3/10/2015 um 3:51 PM schrieb Brandon Invergo:
>> @@ -121,7 +121,7 @@ if (enabled_parameters)
>>     ## seperate FASTA files (usually they destroy the layout)
>>     idx_fastafile = grepl("fasta file", d_parAll$parameter, ignore.case = T)
>>     d_par_file = d_parAll[idx_fastafile, ]
>> -  fasta_files = sapply(unlist(strsplit(d_par_file$value, "\n")), function(x) rev(strsplit(x,"\\", fixed=T)[[1]])[1])
>> +  fasta_files = sapply(unlist(strsplit(d_par_file$value, "\n")), function(x) rev(strsplit(x,.Platform$file.sep, fixed=T)[[1]])[1])
>
> Oh ok, I didn't catch that.  It was running fine with my erroneous replacement so I didn't look further into it.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cbielow/PTXQC/pull/1/files#r26128117
>